### ISSUE TITLE
ToKana not converting space chars

### ISSUE THREAD

**Issue by deelawn** [WaniKani/WanaKana#158]
## Issue Type

Bug

## Description

Calling the `toKana` function doesn't convert `0x20` space characters to the `0x3000` ideographic space character.  I first noticed this when looking at the test cases for the `toKatakana` function.  The test case has `toukyou, おおさか` as the input and `トウキョウ、　オオサカ` as the expected output.

## Steps to reproduce

Call `toKana` with the input from the description above and verify the `0x20` space character is present.

# Versions

v5.1.0

**Comment by DJTB** [comment]
Correct, this is just a typo / mistake in the documentation, sorry about that!

The actual test files cover a lot of cases not mentioned in the basic docs, it's worth taking a dive if you need further clarification.

For example: https://github.com/WaniKani/WanaKana/blob/fix/readme_test_case_typo/test/core/toKana.test.js#L39
